main.go

```diff
@@ -103,9 +105,16 @@ func buildExporter(opts GlobalFlags) *exporter.Exporter {
 
 	log.Debugf("Compatible mode: %v", opts.CompatibleMode)
 
-	if !strings.HasPrefix(opts.URI, "mongodb") {
-		log.Debugf("Prepending mongodb:// to the URI")
-		opts.URI = "mongodb://" + opts.URI
+	if strings.HasPrefix(opts.URI, "mongodb") {
```

Comment:
This breaks existing URI generation logic. Change must be backward compatible. We need to support new way of picking up username and password but still allow old way of providing URI to work.

Reply:
that one was adresed with latest changes